### PR TITLE
feat(docs): update the admin docs to include the dashboard message

### PR DESCRIPTION
**Pull request by leafty**
See: https://github.com/SwissDataScienceCenter/renku-ui/pull/2592.

Note: merge when `renku-ui` is tagged and updated here.

### CODE REVIEW

File: docs/how-to-guides/admin/privacycookie.rst [hunk 1]
```diff
@@ -51,3 +51,29 @@ All the adjustments should be provided as values in ``ui.privacy.banner`` as fol
...
+values in ``ui.client.dashboardMessage``:
+
+1. ``text``: this is the main message displayed on the dashboard page. It supports
+   `Markdown syntax <https://en.wikipedia.org/wiki/Markdown>`_.
+2. ``additionalText``: this is the additional message which users can read by clicking
+   "Read more" just below the main message.
+3. ``styles``: this configures the appearance of the card used to display the
+   dashboard message. Allowed values are: "plain", "success", "info", "warning"
+   and "danger".
+4. ``dissmissible``: when set to true, users can dismiss the message and hide it
```
Comment (lorenzo-cavazzi) [inline docs/how-to-guides/admin/privacycookie.rst:75]: ``dismissible``?

File: docs/how-to-guides/admin/privacycookie.rst [hunk 2]
```diff
@@ -51,3 +51,29 @@ All the adjustments should be provided as values in ``ui.privacy.banner`` as fol
...
+
+Dashboard Message
+~~~~~~~~~~~~~~~~~
+
+The UI can display a configurable message for logged in users on their dashboard
+page, suited for showing general information about Renku.
+
+This feature can be enabled and configured by editing the values found in
+``ui.client.dashboardMessage``. Set ``ui.client.dashboardMessage.enabled``
+to ``true`` to enable have the message displayed on the dashboard page.
```
Comment (lorenzo-cavazzi) [inline docs/how-to-guides/admin/privacycookie.rst:63]: I would drop "have" or change to "to enable _having_ the message displayed".

Or at least Grammarly tells me so :joy:

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Thanks!
Just a typo with "dismissible"; otherwise, ready to go with the next UI release :rocket:

**Review by lorenzo-cavazzi — DISMISSED** [review]
:rocket:

**Review by bethcg — APPROVED** [review]
I would change the name of the page from "Configure the user interface" to "User interface configuration options". It may be a matter of taste.

File: docs/how-to-guides/admin/privacycookie.rst [hunk 3]
```diff
@@ -42,12 +42,38 @@ content. We use `cookie consent <https://github.com/Mastermindzh/react-cookie-co
...
+to ``true`` to have the message displayed on the dashboard page.
+
+The content and appearance of the message are configured by editing the other
+values in ``ui.client.dashboardMessage``:
+
+1. ``text``: the main message displayed on the dashboard page. It supports
+   `Markdown syntax <https://en.wikipedia.org/wiki/Markdown>`_.
+2. ``additionalText``: the additional message which users can read by clicking
+   "Read more" button right below the main message.
+3. ``styles``: the appearance of the card used to display the
```
Comment (rokroskar) [inline docs/how-to-guides/admin/privacycookie.rst:72]: why is this called `styles` and not `style`? only setting one, presumably

Comment (leafty) [reply docs/how-to-guides/admin/privacycookie.rst:72]: it's a typo

**Review by rokroskar — APPROVED** [review]

**Comment by lorenzo-cavazzi** [comment]
Let us merge this; the "dashboard message" feature was released with UI 3.7.0 and merged to master already #3133

**Pull request merged**